### PR TITLE
add columns

### PR DESCRIPTION
**Pull request by dominikkowalski**
add columns to view record request and domain list

### CODE REVIEW

File: ui/static/app/record-request/record-request.component.html [hunk 1]
```diff
@@ -44,6 +46,12 @@
           <td>
             {{ request.target_owner }}
           </td>
+          <td>
+            <a href="{{ jira_url }}" target="_blank"> {{ request.key }} </a>
```
Comment (szok) [inline ui/static/app/record-request/record-request.component.html]: It is ok? 
link must be directed to ticket, not to Jira in global.

File: dnsaas/settings.py [hunk 2]
```diff
@@ -229,6 +229,7 @@ def __getitem__(self, item):
 NEW_FEATURE_URL = ''
 SENTRY_JS_DSN = ''
 SENTRY_JS_CONFIG = {}
+JIRA_URL = ''
```
Comment (szok) [inline dnsaas/settings.py]: Please change name to JIRA_TICKET_URL or something similar

**Comment by szok** [comment]
:ok:

**Comment by krzysztofzygmunt** [comment]
:ok:

**Pull request merged**